lib/config.js

```diff
@@ -27,7 +27,7 @@ try {
 } catch (e) {}
 
 try {
-  require('ts-node').register()
+  require('ts-node')
```

Comment:
I gather you are saying: we don't need to call register because all we have no options?

Reply:
The fact that Karma does this is highly bothersome: https://github.com/karma-runner/karma/issues/3329

Use of `ts-node` is not a bad idea, but it should be optional, with some way to enable it.

Reply:
@trusktr thanks for the example thats exactly the problem I encountered too. You can only call `register()` once in your whole project after that it is not possible to change the `options` given to the `register` call. The `register` call in your `kamra.conf.ts` is simply ignored cause `karma` already called it here. The only way to inject some other configuration is via the `tsconfig.json` which needs to be the top level `tsconfig.json` file. This is not suitable for all projects.

This will be a breaking change cause, if we remove the `register` call we need a `karma.conf.js` that calls `register` for us. Another solution could be to allow the injection of a `tsconfig.json` via a command line arg.

For me I was able to find a `tsconfig.json` configuration that fits `karma` and the rest of the project. But the way to this solution was very cumbersome.

Reply:
Please add the breaking change info

https://www.conventionalcommits.org/en/v1.0.0/

and the doc change.